examples/superstages.py

```diff
@@ -69,7 +69,7 @@
 out = asim.run_aurora(D_z, V_z, times_DV=[1.0,], unstage=True, plot=plot)
 
 # extract densities and particle numbers in each simulation reservoir
-nz, N_wall, N_div, N_pump, N_ret, N_tsu, N_dsu, N_dsul, rcld_rate, rclw_rate = out
```

Comment:
@antonellozito once at this point, I realized that these changes are NOT backwards compatible... users that are unwrapping the `out` output from `run_aurora` will get an unexpected number of elements... Can you think of any way to avoid this?

Reply:
@fsciortino you are right, and this is something we should discuss. Actually another possible backwards incompatibility is having changed the name of the `check_conservation` subroutine into `plot_reservoirs` (but this is simply an easthetical choice and fully revertible).
For `out`, I am not sure how to do it. The main feature of all the changes that I am making is the presence of a larger number of reservoirs, so this must be in the output. If we keep out as a tuple, there is no way to circumvent this, as the number of elements in it will be unavoidably different as before.
One possible way to avoid this is to build `out` as a different python structure with elements which are not ordered anymore, which can be then called by name rather than by index (so even adding 100 more reservoirs in the future, `N_div` can be always invoked just as `N_div`, instead of `out[xyz]`). But this would also affect existing user scripts as they would need to change the way in which they interact with `out`...